src/auth/jwtaccess.ts

```diff
@@ -14,19 +14,25 @@
  * limitations under the License.
  */
 
+import {IncomingHttpHeaders} from 'http';
```

Comment:
I'm wondering if we really need to use `IncomingHttpHeaders` if we already know that the headers are of type `type T = { [key: string]: string }`. If that's true I would suggest using that because `T` is a subtype of `IncomingHttpHeaders` and you don't have to cast values as strings in tests.

Reply:
That's an entirely fair point 👍